pkg/models/log_entry.go

```diff
@@ -16,22 +16,22 @@ package models
 import "encoding/json"
 
 type LogEntry struct {
-	Level         string   `json:"logLevel"`
-	Labels        []string `json:"labels"`
-	OriginService string   `json:"originService"`
-	Message       string   `json:"message"`
-	Created       int64    `json:"created"`
+	Level         string        `json:"logLevel"`
+	KeyVals       []interface{} `json:"labels"`
```

Comment:
You changed `Labels` by `KeyVals`, but in `internal/support/logging/models/log_entry.go` you changed `Labels` by `Args`. In both cases json label is still `labels`
Is that correct?